### ISSUE TITLE
new keras syntax?

### ISSUE THREAD

**Issue by asimkievich** [fchollet/keras#2070]
I have a script that worked just fine in a previous version of keras. Running the script on distribution 0.3.2, I get all sort of error messages. Is there any 'from-to' type of document to go through and be in an easier position to update older keras scripts?

PS: some sample lines of the script are:

```
model = Sequential()
model.add(Dropout(self.dropx[0]))
model.add(Dense(nb_features, self.nb_neuronx[0], init='glorot_uniform'))
model.add(PReLU((self.nb_neuronx[0],)))
```

and one of the errors I get: 

```
Exception: Layer is not connected. Did you forget to set "input_shape"?
```

**Comment by NasenSpray** [comment]
There's no guide, but you can compare with the [documentation](www.keras.io) and the provided examples.

**Comment by fchollet** [comment]
Your dropout layer needs an `input_shape` argument. The Dense layer now
only needs to `output_dim` argument (when it isn't the first layer in a
model). These changes are from September 2015.

**Comment by asimkievich** [comment]
thanks a lot François, for the tip and for putting together an outstanding library.

**Comment by asimkievich** [comment]
hi Françõis, is there a way to retrieve a pre-september 2015 version from the repo? I have been struggling with the new syntax and have a tough deadline, so I prefer to roll-back to the previous version, at least short term. Is that possible?

**Comment by lemuriandezapada** [comment]
if you have the git version, just git timeline and git checkout an earlier commit

**Comment by fchollet** [comment]
Sure, it's available for download from Github and PyPI.

**Comment by asimkievich** [comment]
I may be asking for too much, but could you spell out the exact command that I need to download a pre-September 2015 version via pip? :) thanks a lot! I promise I will learn the new version time permitting!
